File: polyglot/1-variables-and-expressions.ts
let g = 1

function app (): void {
  const x = 42
  let y = 41
  const z = null

  let [a, b] = [1, -1]
  ;[a, b] = [b, a]

  g += 1
  y += 1
  // x += 1

  console.log({ g, x, y, z, a, b })
  console.log(`z is null? ${z === null ? 'true' : 'false'}`)
  console.log(`x is ${(x % 2 === 0) ? 'even' : 'odd'}.`)
}

app()

File: polyglot/1-variables-and-expressions.py
from typing_extensions import Final

g = 1

def app () -> None:
    global g

    X: Final = 42
    y: int = 41
    Z: Final = None

    a, b = 1, -1
    a, b = b, a

    g += 1
    y += 1
    # X += 1

    print(dict(g=g, X=X, y=y, Z=Z, a=a, b=b))
    print(f"Z is None? {Z is None}")
    print(f"X is {'even' if X % 2 == 0 else 'odd'}.")

app()
